apps/studio/pages/project/[ref]/settings/vault/secrets.tsx

```diff
@@ -35,19 +35,22 @@ const VaultSettingsSecrets: NextPageWithLayout = () => {
       ) : !isEnabled ? (
         <VaultToggle />
       ) : (
-        <Tabs
-          size="small"
-          type="underlined"
-          activeId="secrets"
-          onChange={(id: any) => {
-            if (id === 'keys') router.push(`/project/${ref}/settings/vault/keys`)
+        <Tabs_Shadcn_
+          defaultValue="secrets"
+          value={router.query.tab as string}
+          onValueChange={(e) => {
```

Comment:
https://ui.shadcn.com/docs/components/tabs
the tabs here are arrow-key-accessible.

I realized that the Vault page doesn't do this b/c of the router.push. Wonder if router.replace would work / makes sense here?